envs/openssl-env.yaml

```diff
@@ -19,12 +19,13 @@ packages:
     patches :
       - feedstock-patches/rust-activation/0001-Fixed-build.patch
   - feedstock: https://github.com/AnacondaRecipes/cryptography-feedstock
-    git_tag : b6df720b6804150350fd1c268df513ffb8113f26                                #[not ppc64le]
-    patches:                                                                          #[not ppc64le]
-      - feedstock-patches/cryptography/0001-opence-changes-for-crytography.patch      #[not ppc64le]
-    git_tag : 926030e5fd40c00a9716e00efa62f4e286aaa8b1                                #[ppc64le]
-    patches :                                                                         #[ppc64le]
-      - feedstock-patches/cryptography/0001-opence-changes-for-crytography-v39.patch  #[ppc64le]
+    git_tag : 9a5d0096892ec8c0a990026fef0b641160dcfaa1   #v41.0.3
```

Comment:
why does this say `41.0.3`?

Reply:
Actually anaconda channel have crypto=41.0.3 only and we had patched it to use 41.0.4 further